plugins/GSdx/Renderers/OpenGL/GSTextureOGL.cpp

```diff
@@ -150,6 +150,9 @@ namespace PboPool {
 GSTextureOGL::GSTextureOGL(int type, int w, int h, int format, GLuint fbo_read, bool mipmap)
 	: m_clean(false), m_generate_mipmap(true), m_local_buffer(nullptr), m_r_x(0), m_r_y(0), m_r_w(0), m_r_h(0), m_layer(0)
 {
+	ASSERT(w > 0);
+	ASSERT(h > 0);
```

Comment:
I'm starting to wonder if there's any point in these asserts if we're just manually  enforcing them into a range later, also these asserts trigger on release builds or only on dev and lower?

Reply:
Only on debug. Just there to warn developer about potential hidden issue or to warn them about improper new code.